tests/middleware/test_proxy_headers.py

```diff
@@ -1,12 +1,16 @@
-from typing import List, Union
+from __future__ import annotations
```

Comment:
Given that this `__future__` is in somewhat of a limbo, I would be reluctant to merge it.